django/db/models/functions/datetime.py

```diff
@@ -223,6 +223,19 @@ def as_postgresql(self, compiler, connection, **extra_context):
             compiler, connection, template="STATEMENT_TIMESTAMP()", **extra_context
         )
 
+    def as_mysql(self, compiler, connection, **extra_context):
+        return self.as_sql(
+            compiler, connection, template="CURRENT_TIMESTAMP(6)", **extra_context
+        )
+
+    def as_sqlite(self, compiler, connection, **extra_context):
+        return self.as_sql(
+            compiler,
+            connection,
+            template="STRFTIME('%%Y-%%m-%%d %%H:%%M:%%f', 'NOW')",
```

Comment:
@felixxm These needed to be `%%%%`: https://docs.djangoproject.com/en/4.1/ref/models/expressions/#django.db.models.Func.template

I'm looking into writing a standalone test that proves this, but it is currently causing the sqlite tests to break on #16092.

Reply:
Actually, maybe this is actually a bug on the db defaults side...

Reply:
I missed that it's documented, however both versions works on SQLite as it has a custom `SQLiteCursorWrapper`:
https://github.com/django/django/blob/649b28eab6765cd6b2b40c779a18ecafc99b43d7/django/db/backends/sqlite3/base.py#L355-L376 
and the format string doesn't contain `%s`. I'm not sure if it's worth changing :thinking: